apis/youtube/api.go

```diff
@@ -0,0 +1,94 @@
+package youtube
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+)
+
+const (
+	part   = "snippet"
+	fields = "items.id,items.snippet.title,items.snippet.description,items.snippet.publishedAt"
```

Comment:
I believe it would be better (and probably safer) to have this fields list as a string array, and then join them together when necessary: 
```go
	fields = []string{"items.id", "items.snippet.title", "items.snippet.description", "items.snippet.publishedAt"}
```

Then when they need to be joined we can use
```go
strings.Join(fields, ",")
```

Reply:
Unfortunately, golang doesn't support const array. We can build a function `getFields`, instead. What do you think?

Reply:
Oh, yeah, you are right. Fixed it.